tests/e2e/commands/subnet.go

```diff
@@ -396,7 +396,7 @@ func DeploySubnetLocallyWithArgsSOV(subnetName string, version string, confPath
 	gomega.Expect(exists).Should(gomega.BeTrue())
 
 	// Deploy subnet locally
-	cmdArgs := []string{SubnetCmd, "deploy", "--local", subnetName, "--" + constants.SkipUpdateFlag, bootstrapFilepathFlag + "=" + utils.BootstrapValidatorPath}
+	cmdArgs := []string{SubnetCmd, "deploy", "--local", subnetName, "--" + constants.SkipUpdateFlag}
```

Comment:
Since now using `bootstrapFilepathFlag` flag -> convert only & no local machine spun up, this flag will be removed in existing e2e `blockchain deploy` tests

Reply:
`bootstrapFilepathFlag` is now tested in [tests/e2e/commandse2e/blockchain/deploy/suite.go](https://github.com/ava-labs/avalanche-cli/pull/2788/files#diff-b6718681fd2c8c523bd1c7eefde681118249ae62aacd71e41bc933a0b36ee742)